pyproject.toml

```diff
@@ -19,7 +19,6 @@ include = '\.pyi?$'
 
 [tool.pytest.ini_options]
 addopts = """
-    -n auto
```

Comment:
- [ ] Put this back. I just have it off for debugging purposes. This actually helps a lot for these tests though.